diff --git a/node_modules/twitter-text/dist/esm/index.js b/node_modules/twitter-text/dist/esm/index.js
index 4914f5d..9eb86dd 100644
--- a/node_modules/twitter-text/dist/esm/index.js
+++ b/node_modules/twitter-text/dist/esm/index.js
@@ -28,6 +28,7 @@ import hasInvalidCharacters from './hasInvalidCharacters';
 import hitHighlight from './hitHighlight';
 import htmlEscape from './htmlEscape';
 import isInvalidTweet from './isInvalidTweet';
+import isValidCashtag from './isValidCashtag';
 import isValidHashtag from './isValidHashtag';
 import isValidList from './isValidList';
 import isValidTweetText from './isValidTweetText';
@@ -76,6 +77,7 @@ export default {
   hitHighlight: hitHighlight,
   htmlEscape: htmlEscape,
   isInvalidTweet: isInvalidTweet,
+  isValidCashtag: isValidCashtag,
   isValidHashtag: isValidHashtag,
   isValidList: isValidList,
   isValidTweetText: isValidTweetText,
diff --git a/node_modules/twitter-text/dist/esm/isValidCashtag.js b/node_modules/twitter-text/dist/esm/isValidCashtag.js
new file mode 100755
index 0000000..2c3bd5e
--- /dev/null
+++ b/node_modules/twitter-text/dist/esm/isValidCashtag.js
@@ -0,0 +1,13 @@
+// Copyright 2018 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+import extractCashtags from './extractCashtags';
+export default function (cashtag) {
+  if (!cashtag) {
+    return false;
+  }
+
+  var extracted = extractCashtags(cashtag); // Should extract the cashtag minus the $ sign, hence the .slice(1)
+
+  return extracted.length === 1 && extracted[0] === cashtag.slice(1);
+}
\ No newline at end of file
